core/Cargo.toml

```diff
@@ -20,5 +20,6 @@ tokio-stream = "0.1.9"
 tracing = "0.1.36"
 tracing-stackdriver = "0.5.0"
 tracing-subscriber = { version = "0.3.15", features = ["env-filter"] }
+trust-dns-server = "0.22.0"
```

Comment:
Rather than add this dependency here, let's just do anything that uses trust dns types in the DNS server itself. The advantages are:
- It reduces the number of dependencies that need to be fetched and built by consumers of this library who just need the message schemas.
- It avoids tying us to a particular library at the Rust interface level, making the choice of a particular library an implementation decision.

It was a mistake on my part to make this crate depend on bollard, and I'd like to refactor that dependency out as part of #137.

Reply:
I was thinking this, but didn't feel like implementing display (for format) and figured it'd be fine since bollard was in there too. yea makes sense to me I'll drop it.

Reply:
Yeah, I appreciate that you tried to do it in a way that was consistent with how we do it for bollard. It was the right design decision given the context you had; my bad for not giving you the context that I want to remove bollard.

Reply:
rm this now that it's not used